scripts/shortcuts/review/review.sh

```
@@ -38,18 +38,33 @@ done
+template="$here/prompts/review.md"
+if [ ! -f "$template" ]; then
+  echo "[review] missing prompt template: $template" >&2
+  exit 1
+fi
+
+if [ "${REVIEW_HAS_CONFLICTS:-0}" = "1" ]; then
+  conflict_block="# ⚠️ Merge conflicts detected
+
+When the PR branch was merged with current \`main\`, the following files were left with unresolved conflict markers:
+
+$(printf -- '- %s\n' $REVIEW_CONFLICT_FILES)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)